src/EnergyPlus/DaylightingManager.cc

```diff
@@ -4272,7 +4272,7 @@ void GetDaylightingParametersInput(EnergyPlusData &state)
             bEndofErrFile = !iDElightErrorFile.good();
             bRecordsOnErrFile = false;
             while (!bEndofErrFile) {
-                auto cErrorLine = iDElightErrorFile.readLine();
+		auto cErrorLine = iDElightErrorFile.readLine(); // (THIS_AUTO_OK)
```

Comment:
This looks like leading tabs instead of spaces, drap cursor over this green space

Reply:
Yes, I just ran `apply_formatting.sh`. Did not know how to do that previously.